MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -609,6 +609,11 @@
 				}
 				break;
 			}
+			case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_INDEX_TYPE_UINT8_FEATURES_EXT: {
+				auto* indexTypeUint8Features = (VkPhysicalDeviceIndexTypeUint8FeaturesEXT*)next;
+				indexTypeUint8Features->indexTypeUint8 = true;
+				break;
+			}
```

Comment:
To support both `KHR`, `EXT`, and future 1.4, suggest you remove `EXT` suffixes and move this to the appropriate sorted position in the main non-extension area above.